package.json

```diff
@@ -23,19 +26,19 @@
     "type": "git",
     "url": "git+https://github.com/strongloop/loopback4-example-log-extension.git"
   },
-  "keywords": [
-    "loopback4",
-    "lb4ext",
-    "typescript",
-    "loopback-extension"
-  ],
+  "keywords": ["loopback4", "lb4ext", "typescript", "loopback-extension"],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/strongloop/loopback4-example-log-extension/issues"
   },
   "homepage": "https://github.com/strongloop/loopback4-example-log-extension#readme",
```

Comment:
https://github.com/strongloop/loopback4-example-log-extension?

Reply:
It's the same page that loads but anchored to start at the README rather than the top of the Repo page. This was the default from `npm init`. I personally prefer it but don't mind switching it to just the repo link.

Reply:
Fine with me then to keep it as is.